configurator/src/Configurator/Vendor/Keycloak/KeycloakConfigurator.php

```diff
@@ -163,6 +163,40 @@ public function configure(OutputInterface $output, array $presets): void
         }
     }
 
+    public function synchronize()
+    {
+        $this->configureRealm();
+
+        foreach ($this->frontendApplications as $app) {
+            $baseUri = getenv(sprintf('%s_CLIENT_URL', strtoupper($app)));
+
+            $this->keycloakManager->updateClientByClientId(
+                getenv(sprintf('%s_CLIENT_ID', strtoupper($app))),
+                [
+                    'rootUrl' => $baseUri,
+                    'redirectUris' => $baseUri ? [$baseUri.'/*'] : null,
+                    'webOrigins' => [$baseUri]
+                ]
+            );
+        }
+
+        foreach ($this->symfonyApplications as $app) {
+            $baseUri = getenv(sprintf('%s_API_URL', strtoupper($app)));
+
+            $this->keycloakManager->updateClientByClientId(
+                getenv(sprintf('%s_ADMIN_CLIENT_ID', strtoupper($app))),
+                [
+                    'rootUrl' => $baseUri,
+                    'redirectUris' => [
+                        $baseUri.'/admin/*',
+                        $baseUri.'/bundles/apiplatform/swagger-ui/oauth2-redirect.html',
+                    ],
+                    'webOrigins' => [$baseUri]
+                ]
+            );
+        }
```

Comment:
Can you reuse existing code in `App\Configurator\Vendor\Keycloak\KeycloakConfigurator::configure` method by extracting a `configureClients` method?